gfx/portraits/trait_portrait_modifiers/00_trait_modifiers.txt

```diff
@@ -1204,3 +1204,28 @@ drunkard = {
 		}
 	}
 }
+
+magic_good = {
+	magic_good = {
+		trigger= {
+			# Warcraft
+			has_no_portrait_trigger = no	
+			has_trait = creature_night_elf
+		}
+		traits = {
+			magic_good_1
+			magic_good_2
+			magic_good_3
+			magic_good_noninheritable
+			}
+		dna_modifiers = {
+			color = {
+			gene = eye_color
+			mode = replace
+			x = 0.11
+			y = 0.3
```

Comment:
The current color feels too orange to me.
```suggestion
			x = 0.19
			y = 0.5
```

Reply:
Couldn't add in batch commit, so I will add this manually into next commit. Looking at other pictures of in game WoW characters, I agree. I thought maybe they were more amber, but they are really much more yellow.